.ado/pipelines/templates/jobs-workload-deploy.yaml

```diff
@@ -74,18 +107,15 @@ jobs: # using jobs so they each run in parallel
           # load json data from downloaded terraform artifact
           $releaseUnitInfraDeployOutput = Get-ChildItem $(Pipeline.Workspace)/terraformOutputReleaseUnitInfra/*.json | Get-Content | ConvertFrom-JSON
 
-          echo "*** Retrieved Azure Front Door Header ID $(frontdoor_id_header) from Terraform output"
```

Comment:
@sebader was it intended to drop this? I think this is valuable debug output.

Reply:
yeah I realized we never really needed this one (or never had issues). you can always just look it up in the artifact. and it is kind of a secret so not logging it is probably better